webdriver/tests/bidi/script/add_preload_script/arguments.py

```diff
@@ -21,7 +21,7 @@
                         "baz",
                         {
                             "type": "object",
-                            "value": [["1", {"type": "number", "value": 2}]],
+                            "value": [["qux", {"type": "number", "value": 2}]],
```

Comment:
Why change this? We definitely need to support string values that look like numbers, so  I’m not sure why we’d avoid this in a test.

Reply:
To unscreen the preload tests. I added the tests for serialization.

Reply:
What does “unscreen” mean?